freemocap/core_processes/capture_volume_calibration/anipose_camera_calibration/anipose_camera_calibrator.py

```diff
@@ -93,21 +93,22 @@ def calibrate_camera_capture_volume(self, pin_camera_0_to_origin: bool = False):
         calibration_folder_toml_path = Path(get_calibrations_folder_path()) / calibration_toml_filename
 
         self._anipose_camera_group_object.dump(calibration_folder_toml_path)
-        logger.info(
+        logger.debug(
             f"anipose camera calibration data saved to Calibrations folder - {str(calibration_folder_toml_path)}"
         )
 
         recording_folder_toml_path = Path(self._recording_folder_path) / calibration_toml_filename
 
         self._anipose_camera_group_object.dump(recording_folder_toml_path)
-        logger.info(f"anipose camera calibration data saved to Recording folder - {str(recording_folder_toml_path)}")
+        logger.debug(f"anipose camera calibration data saved to Recording folder - {str(recording_folder_toml_path)}")
 
         last_successful_calibration_toml_path = get_last_successful_calibration_toml_path()
         self._anipose_camera_group_object.dump(last_successful_calibration_toml_path)
 
-        logger.info(
+        logger.debug(
```

Comment:
```suggestion
        logger.info(
```

I think as a general rule, if we should show saving something to disk, we should show the path to the User

Reply:
(just say that you might be doing his in the `.._progress_callback` below - Your call!

Reply:
I think the progress callback is good enough here - remember the debug messages will still be saved in the logs and printed to the terminal. Also generally, the calibration toml stuff gets logged in a few different places